donation-api/src/main/java/org/interfaithsanctuary/donationapi/model/Callout.java

```diff
@@ -48,51 +42,35 @@ public void setTitle(String title) {
         this.title = title;
     }
 
-    public String getBody() {
-        return body;
-    }
-
-    public void setBody(String body) {
-        this.body = body;
-    }
-
-    public String getUrl() {
-        return url;
-    }
-
-    public void setUrl(String url) {
-        this.url = url;
-    }
-
-    public Date getCreateDate() {
-        return createDate;
+    public Date getEffectiveDate() {
+        return effectiveDate;
     }
 
-    public void setCreateDate(Date createDate) {
-        this.createDate = createDate;
+    public void setEffectiveDate(Date effectiveDate) {
+        this.effectiveDate = effectiveDate;
     }
 
-    public Date getUpdateDate() {
-        return updateDate;
+    public boolean isActive() {
+        return active;
```

Comment:
Is isActive a better variable name since it conveys that the field is a boolean more clearly?